src/lib/schematics/utils/build-component.ts

```diff
@@ -24,40 +24,51 @@ import {
   url,
 } from '@angular-devkit/schematics';
 import {FileSystemSchematicContext} from '@angular-devkit/schematics/tools';
-import * as ts from 'typescript';
-import {addDeclarationToModule, addExportToModule} from './ast-utils';
-import {InsertChange} from './change';
-import {buildRelativePath, findModuleFromOptions} from './find-module';
-import {getWorkspace} from './config';
-import {parseName} from './parse-name';
-import {validateName} from './validation';
+import {Schema as ComponentOptions} from '@schematics/angular/component/schema';
```

Comment:
Is everything under `devkit-utils` still copied exactly from the devkit source?

Reply:
Our `devkit-utils` have been removed in favor of the *same* devkit utils that can be just imported. This also makes upgrades to more recent Angular CLI versions way more maintainable.

Reply:
What I mean, though, is that formerly *everything* under devkit-utils was copied verbatim. If we're not copying _anything_ any more, I'd like to drop that directory in favor of one or more directories that are more specifically named.

Reply:
I see. I'd prefer having another cleanup-run in a follow-up. This PR is mostly about the testing, and switching to the latest devkit utils was just a necessary step in order to work with 6.0.0

Reply:
Yeah, that sounds good to me